TourMate/TourMate/Frontend/Plan/Services/FirebasePlanService.swift

```diff
@@ -56,6 +56,11 @@ class FirebasePlanService: PlanService {
         return (plans, errorMessage)
     }
 
+    func deleteSinglePlan(plan: Plan) async -> (Bool, String) {
+        print("[FirebasePlanService] Deleting single plan")
+        return await planRepository.deleteItem(id: plan.versionedId)
+    }
```

Comment:
We most likely won't use this if we keeping all the versions right

Reply:
Yup, will remove